examples/enable-operator-and-auto-instrumentation/README.md

```diff
@@ -0,0 +1,138 @@
+# Example of chart configuration
+
+## How to deploy the OpenTelemetry Operator and Java auto-instrumentation
+
+In the following example we will show how to instrument a project based on
+[spring-petclinic-microservices](https://github.com/spring-petclinic/spring-petclinic-microservices).
+
+## Getting started
```

Comment:
Do we really need 3 "getting started" pages? :)

Reply:
This me being an engineer who writes docs, willl update.

Reply:
Done